src/components/panel/TextViewsToggle.tsx

```diff
@@ -65,7 +71,7 @@ const TextViewsToggle: FC = () => {
                     onClick={(e) => handleTextViewClick(e, button.view)}
                     variant={panelState.view === button.view ? 'secondary' : 'ghost'}
                     size="icon"
-                    disabled={!panelState.contentTypes?.length}
+                    disabled={!panelState.contentTypes?.length || !isEnabled(button.view)}
```

Comment:
This should also into isEnabled, so the function becomes a central place to determine whether a button is enabled or not